step-release-vis/src/app/components/environments/environments.ts

```diff
@@ -123,7 +123,7 @@ export class EnvironmentsComponent implements OnInit {
    */
   private refresh(assignNewColors = true): void {
     this.setStartEndTimestamps();
-    this.updateDimensions(window.innerWidth, window.innerHeight);
+    this.updateDimensions(document.body.clientWidth, window.innerHeight);
```

Comment:
waht if we have a scrollbar to the other direction for any reason? let's fix it for both dimensions, and add a test ;)

Reply:
I have absolute positions vertically so elements do not take up space in the body, therefore the client height is zero